app/modules/media/media_service.py

```
@@ -0,0 +1,624 @@
+    async def get_attachment(self, attachment_id: str) -> Optional[MessageAttachment]:
+        """Get attachment record by ID"""
+        try:
+            attachment = (
+                self.db.query(MessageAttachment)
+                .filter(MessageAttachment.id == attachment_id)
+                .first()
+            )
+            return attachment
+        except Exception as e:
+            logger.error(f"Error retrieving attachment {attachment_id}: {str(e)}")
+            raise MediaServiceError(f"Failed to retrieve attachment: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)